frontend/src/components/aeppslist.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+const list = [
```

Comment:
Change this to something meaningful. **Alway give variables meaningful names**.
e.g. aeppList